modules/tracktion_engine/tracktion_engine.h

```diff
@@ -34,7 +34,8 @@
 #pragma once
 #define TRACKTION_ENGINE_H_INCLUDED
 
-#if ! JUCE_MODAL_LOOPS_PERMITTED
+// Android does not support modal loops.
+#if ! JUCE_MODAL_LOOPS_PERMITTED && ! JUCE_ANDROID
```

Comment:
And Tracktion compiles okay with this flag? That seems a little odd, but I guess if it works, it works!

Reply:
Yep! It's actually what Dave Rowland suggested to do. He mentioned the new tracktion engine 2.0 won't need this.